kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/AdminClientServiceImpl.java

```diff
@@ -32,11 +37,14 @@
 
   private Mono<ReactiveAdminClient> createAdminClient(KafkaCluster cluster) {
     return Mono.fromSupplier(() -> {
+
       Properties properties = new Properties();
       properties.putAll(cluster.getProperties());
-      properties
-          .put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, cluster.getBootstrapServers());
+
+      properties.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, cluster.getBootstrapServers());
       properties.put(AdminClientConfig.REQUEST_TIMEOUT_MS_CONFIG, clientTimeout);
+      properties.putAll(kafkaSslManager.getSslProperties());
```

Comment:
Why do we need this special config? Could we just push these params?

Reply:
Discussed, found the problem, turns out it does work with properties.